timeboost/src/lib.rs

```diff
@@ -74,8 +87,12 @@ impl Timeboost {
                     }
                 },
                 trx = self.sequencer.next_transactions() => match trx {
-                    Ok(trx) => {
+                    Ok((trx, r, t)) => {
                         info!(node = %self.label, len = %trx.len(), "next batch of transactions");
+                        if let Some(ref mut f) = self.nitro_forwarder {
+                            let d = Data::encode(r, t, &trx)?;
+                            let _ = f.send(d).await;
```

Comment:
Maybe the actual sending should be asynchronous and the data item should only be enqueued here.